app.rb

```diff
@@ -16,31 +16,41 @@ def initialize
   end
 
   def list_people
-    @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"}
+    if @people == []
+      puts
+      puts "No people in system."
+    else
+      @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"}
+    end
```

Comment:
- [ ] After creating the teacher successfully, it is not being displayed. Kindly fix this bug too so as to maintain the functionality of the original design👍
- Your console app:
![Annotation 2022-05-19 043713](https://user-images.githubusercontent.com/85113567/169199883-e1e0c37d-4da3-4e47-977b-cfdf77a175b3.jpg)